app/screens/home/search/initial/modifiers/index.tsx

```diff
@@ -8,7 +8,8 @@ import Animated, {useSharedValue, useAnimatedStyle, withTiming} from 'react-nati
 
 import FormattedText from '@components/formatted_text';
 import {useTheme} from '@context/theme';
-import TeamPickerIcon from '@screens/home/search/team_picker_icon';
+import {TeamModel} from '@database/models/server';
+import TeamPickerIcon from '@screens/home/search/team_picker_icon/team_picker_icon';
```

Comment:
If we are not using anymore the index file to get the information from the database, either rename the file to `index.ts` or even better, remove the folder and put the file in the parent directory (`screens/home/search`).